fendermint/ipc/src/message.rs

```diff
@@ -0,0 +1,19 @@
+// Copyright 2022-2023 Protocol Labs
+// SPDX-License-Identifier: Apache-2.0, MIT
+
+//! IPC messages
+
+use crate::pof::IPCParentFinality;
+use serde::{Deserialize, Serialize};
+
+#[derive(Debug, Clone, Serialize, Deserialize, PartialEq, Eq)]
+pub enum IPCMessage {
```

Comment:
Let's move the IPC related messages (and messages only) into the `fendermint_vm_message` crate, so that doesn't have to include references to crates with implementation such as the IPC Agent proxy. It should be a crate that anyone can add as a dependency to be able to inspect chain messages, until we figure out a way to make them more generic.

Reply:
On a side note: I always struggle to decide between using acronyms in PascalCasing names, but I usually go with only capitalising the first letter, e.g. `ProductId`, `XmlHttpRequest`, `JsonRpcServer`, rather than `ProductID`, `XMLHTTPRequest`, `JSONRPCServer`. I see there are all sorts of variations out there, but I found this to be a perhaps less aesthetically pleasing, but at least consistent way. 

Have you guys established the `IPC*` prefix?

Reply:
I added the equivalent of this enum in the crate I mentioned above in https://github.com/consensus-shipyard/fendermint/pull/187 with a placeholder for `TopDown`.